assets/state-vfio-manager/0400_configmap.yaml

```diff
@@ -123,6 +125,44 @@ data:
         fi
     }
 
+    get_graphics_aux_dev() {
+        local gpu=$1
+        device_class_file=$(readlink -f "/sys/bus/pci/devices/$gpu/class")
+        device_class=$(cat "$device_class_file")
+        if [ "$device_class" == "0x030000" ]; then
```

Comment:
For better readability, I would suggest inverting this if statement and returning early if the device is not a VGA controller:
```
if [ "$device_class" != "0x030000" ]; then
  echo "NONE"
  return
fi

if ls "/sys/bus/pci/devices/$gpu" | grep consumer >& /dev/null; then
   <snip>
fi
<snip>

```